odata-protocol/11.4 Data Modification.md

```diff
@@ -314,8 +325,8 @@ On failure, the service MUST NOT create any of the entities.
 
 ### ##subsubsec Update an Entity
 
-To update an individual entity, the client makes a `PATCH` or `PUT`
-request to a URL that identifies the entity. The [edit URL](#ReadURLsandEditURLs)
+To make changes to an entity, the client makes `PATCH` or `PUT`
+request (a so-called _upsert request_) to a URL that identifies the entity. The [edit URL](#ReadURLsandEditURLs)
```

Comment:
I would not use upsert here; depending on whether the if-match it may just be an update.